### ISSUE TITLE
ng-change fires for every item on page init

### ISSUE THREAD

**Issue by fokoenecke** [angular-ui/ui-select#449]
Hi :3,

i am using ui-select in an ng-repeat, posting the updated item to my server, basically like this:
```html
<div ng-repeat="item in items" >
	<ui-select multiple id="itemChildren{{item.id}}"
						ng-model="item.children_ids"
						reset-search-input="true"
						ng-change="updateItem(item)"
						ng-disabled="disabled">

		<ui-select-match placeholder="Choose Children...">{{$item.name}}</ui-select-match>
		<ui-select-choices repeat="child.id as child in children | filter: $select.search">
			<span>{{child.name}}</span>
		</ui-select-choices>
	</ui-select>
</div>
```
Both `items` and `children` are set via `ng-init`. On page load the function `updateItem()` gets triggered for every `item`. I had a look at the tests and stumbled upon `"should change viewvalue only once when updating modelvalue"`. It unfortunately doesn't cover my case. Could anyone tell me if this behaviour is intentionally?

**edit:** the event also gets triggered if I apply and then remove a filter (so the element is hidden and then displayed again).

Greets!

**Comment by wesleycho** [comment]
Closing as a support issue - feel free to open a new issue with reproduction in Plunker & clearer explanation if this is an actual issue with the library.
